### ISSUE TITLE
How do you ensure that the images generated by the same prompt belong to the training set or the test set?

### ISSUE THREAD

**Issue by ctxya1207** [37s/MoE-AGIQA#1]

**Comment by 37s** [comment]
Thank you for your question. The uploaded .mat file ensures that the images generated by the same prompt belong to the training set or the test set.

**Comment by ctxya1207** [comment]
> Thank you for your question. The uploaded .mat file ensures that the images generated by the same prompt belong to the training set or the test set.

I want to ask how you achieved this.

**Comment by ctxya1207** [comment]
How do you divide your training and test sets?
